content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -68,47 +72,162 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
+   
+      * For the `.spec.ClusterIP` field, the control plane records the IP address that is from the same address family as the first service cluster IP range. 
+      * On a single-stack cluster, the `.spec.ClusterIPs` and `.spec.ClusterIP` fields both only list one address. 
+      * On a cluster with dual-stack enabled, specifying `RequireDualStack` in `.spec.ipFamilyPolicy` behaves the same as `PreferDualStack`.
 
-{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+{{< codenew file="service/networking/dual-stack-preferred-svc.yaml" >}}
 
-For comparison, the following Service specification will be assigned an IPv4 address (also known as a "cluster IP") from the configured `service-cluster-ip-range` to this Service.
+   3. This Service specification explicitly defines `IPv6` and `IPv4` in `.spec.ipFamilies` as well as defining `PreferDualStack` in `.spec.ipFamilyPolicy`. Kubernetes will assign an IPv6 and IPv4 address in `.spec.ClusterIPs` and `.spec.ClusterIP` will be set to the IPv6 address because it is the first element in the `.spec.ClusterIPs` array, overriding the default. 
 
-{{< codenew file="service/networking/dual-stack-ipv4-svc.yaml" >}}
+{{< codenew file="service/networking/dual-stack-preferred-ipfamilies-svc.yaml" >}}
 
-### Type LoadBalancer
+#### Dual-stack defaults on existing Services
 
-On cloud providers which support IPv6 enabled external load balancers, setting the `type` field to `LoadBalancer` in additional to setting `ipFamily` field to `IPv6` provisions a cloud load balancer for your Service.
+These examples demonstrate the default behavior when dual-stack is newly enabled on a cluster where Services already exist.
 
-## Egress Traffic
+   1. When dual-stack is enabled on a cluster, existing Services (whether `IPv4` or `IPv6`) are configured by the control plane to set `.spec.ipFamilyPolicy` to `SingleStack` and set `.spec.ipFamilies` to the address family of the existing Service. The existing Service cluster IP will be stored in `.spec.ClusterIPs`. 
 
-The use of publicly routable and non-publicly routable IPv6 address blocks is acceptable provided the underlying {{< glossary_tooltip text="CNI" term_id="cni" >}} provider is able to implement the transport. If you have a Pod that uses non-publicly routable IPv6 and want that Pod to reach off-cluster destinations (eg. the public Internet), you must set up IP masquerading for the egress traffic and any replies. The [ip-masq-agent](https://github.com/kubernetes-sigs/ip-masq-agent) is dual-stack aware, so you can use ip-masq-agent for IP masquerading on dual-stack clusters.
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
 
-## Known Issues
+   You can validate this behavior by using kubectl to inspect an existing service.
+
+```yaml
+kubectl get svc my-service -o yaml
```

Comment:
BTW this line isn't YAML. See https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output for how the style guide recommends showing commands and their output.